checkoutPage.html

```diff
@@ -4,75 +4,115 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="assets/css/hunterstyle.css">
+    <link rel="stylesheet" href="style.css">
```

Comment:
make sure your style.css goes inside assets/css/
so it's like assets/css/style.css